reviewdog.yml

```diff
@@ -0,0 +1,3 @@
+runner:
+  golint:
+    cmd: golint ./...
```

Comment:
You're right, I think we can just plug the whole `golint` command here and just run `reviewdog` directly. I'm not sure why its doc shows the usage as `golint | reviewdog`, since if you don't specify `golint` somewhere in the `reviewdog` options it complains.